git-publish

```diff
@@ -553,9 +553,15 @@ def main():
         tag(tag_name_staging(topic), tag_message, annotate=anno, force=True)
 
     prefix = options.prefix
-    if prefix is None:
+    if prefix is not None:
+        # when prefix == "", it will clear per-branch prefix
```

Comment:
If so, I would prefer to have one "--clear-subject-prefix" since at least that will not need user to remember a long git config name.

But I think we should still consider the case when user specify "--subject-prefix" as an empty string, since we cannot forbid a user to do so. How about we both provide the --clear-subject-prefix and allow to clear the prefix if --subject-prefix is empty? But we can keep the codes not taking "" as a special case.

I will try to push another one soon, and see whether you like it. :)